src/contrib/mlir/runtime/cpu/cpu_runtime.cpp

```diff
@@ -53,31 +53,38 @@ static llvm::cl::opt<std::string>
     clObjectFilename("ngraph-mlir-object-filename",
                      llvm::cl::desc("Dump MLIR JITted-compiled object to file jitted_mlir.o"));
 
-void MLIRCPURuntime::run(void* args)
+void MLIRCPURuntime::run(void* args,
```

Comment:
Better if change this to capture a vector of structs instead: `run(vector<MemRefArg> args)`? 
```
struct MemRefArg{
void* m_tensor; 
vector<size_t> m_shape; 
vector<size_t> m_strides;
};